### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by SteffenBernd** [juanluisbaptiste/docker-otrs#121] [General]
Hello,
i´ve tryed to install this docker but somehow it does not work. 
is there anyone here who could walk me trough an installation?

im getting this after starting up the docker-compose:

![image](https://user-images.githubusercontent.com/92783384/162384763-1d1ec88d-2b10-4dbf-a0c0-71b01c08f1d1.png)

**Comment by juanluisbaptiste** [comment]
@SteffenBernd please post the docker-compose output.

**Comment by SteffenBernd** [comment]
Hello,
first its getting stuck on the mariadb part
![image](https://user-images.githubusercontent.com/92783384/163115039-c944586d-d161-4f08-b22c-619bca7936ff.png)


and then it gets stuck at the otrs_1 part and times out over and over again.

is that what your are looking for? or what do you mean with docker-compose output?

istn there any installation documentation i can follow? the documentation in this git is not very helpfull for someone who is not a pro with linux / docker / compose.

**Reply by juanluisbaptiste** [reply]
> Hello, first its getting stuck on the mariadb part ![image](https://user-images.githubusercontent.com/92783384/163115039-c944586d-d161-4f08-b22c-619bca7936ff.png)
> 
> and then it gets stuck at the otrs_1 part and times out over and over again.
> 

It seems you have write permssions issues with the mariadb volumes.

> is that what your are looking for? or what do you mean with docker-compose output?
> 
> istn there any installation documentation i can follow? the documentation in this git is not very helpfull for someone who is not a pro with linux / docker / compose.

This project assumes you already know how to use docker and docker-compose, if you don't then you should start by learning those instead first before trying out this project.

**Reply by SteffenBernd** [reply]
well i´ve used both before. with a proper installation guide i had no problem. but this has no proper installation guide.

thats why im asking if there is anyone who could guide me through it or has a proper installation guide somewhere

**Comment by juanluisbaptiste** [comment]
There is documentation, see the README file, there is nothing more to it than what it says there, just adjust your .env file and run docker-compose up.
